### ISSUE TITLE
Crashing because of non-exististent book

### ISSUE THREAD

**Issue by kasbah** [Bristol-Braille/canute-ui#26]
Not  exactly sure how it happened, we have been trying to find the issue with the buttons not responding,  but on MK10 I found the UI in a crashed state with: 

```
2016-10-01 21:37:12,757 - root             - INFO     - despatching ui->load_book(4)
2016-10-01 21:37:13,309 - driver_pi        - ERROR    - OSError : [Errno 2] No such file or directory: '/home/pi/books/randomsc3.canute'
2016-10-01 21:37:13,321 - driver_pi        - ERROR    - closing serial port
```

**Comment by kasbah** [comment]
I have mitigated this for the time being by letting OSError's and IOErrors pass with https://github.com/Bristol-Braille/canute-ui/commit/eba2bb59e00a0acca4483d2ea89ecf375013bb4d.

**Comment by kasbah** [comment]
This sort of thing should be fixed with #27
